src/RepoAuditor/CommandLineProcessor.py

```diff
@@ -36,55 +36,35 @@ def __call__(  # noqa: D102
             ],  # actual type is dict[str, TyperEx.TypeDefinitionItemType]
         ) -> dict[str, Any]: ...
 
-    # ----------------------------------------------------------------------
-    # |
-    # |  Public Data
-    # |
-    # ----------------------------------------------------------------------
-    module_infos: list[ModuleInfo]
-    warnings_as_error_module_names: set[str]
-    ignore_warnings_module_names: set[str]
-    single_threaded: bool = field(kw_only=True)
-
-    # ----------------------------------------------------------------------
-    # |
-    # |  Public Methods
-    # |
-    # ----------------------------------------------------------------------
-    @classmethod
-    def Create(
-        cls,
-        get_dynamic_args_func: "CommandLineProcessor.GetDynamicArgsFunc",
+    @staticmethod
+    def GetModuleMap(

```

Comment:
I don't think that we would ever want anyone to call these new functions, therefore indicate that they are private by giving them an underscore prefix (e.g `_GetModuleMap`).